### PR TITLE
Knative Backstage security blogpost

### PR DESCRIPTION
**Pull request by aliok**
"Fixes #issue-number" or "Add description of the problem this PR solves"

## Proposed Changes 

-
-
-

### CODE REVIEW

**Comment by aliok** [comment]
Preview: https://deploy-preview-6126--knative.netlify.app/blog/articles/knative-backstage-security/

File: blog/docs/articles/knative-backstage-security.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+The token is stored in Backstage configuration and is passed to the backend with each request. The backend uses this token to authenticate to the Kubernetes API server. The token is a service account token that has the necessary permissions to list the Knative Eventing resources in the cluster.
+
+```yaml
+...
+catalog:
+  providers:
+    knativeEventMesh:
+      dev:
+        token: "eyJhb.........NHA"
```
Comment (matzew) [inline blog/docs/articles/knative-backstage-security.md:32]: I like `token: '${KNATIVE_EVENT_MESH_TOKEN}' ` better (as in the README).

I'd also add a note here, to not hard-code anything.

Comment (aliok) [reply blog/docs/articles/knative-backstage-security.md]: @matzew new look?

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**